src/components/shared/DeviceProtectedRoute.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Route } from 'react-router-dom';
+import LogoImg from '../../images/susi-logo.svg';
+import styled from 'styled-components';
+import _Button from '@material-ui/core/Button';
+import urls from '../../utils/urls';
+
+const Button = styled(_Button)`
+  width: 332px;
+  margin-top: 2rem;
+  @media (max-width: 356px) {
+    width: 100%;
+  }
+`;
+
+const SusiLogo = styled.img.attrs({
+  alt: 'Page Not Found',
+  src: LogoImg,
+})`
+  width: 15rem;
+  margin-bottom: 2rem;
+`;
+
+const Container = styled.div`
+  min-height: 74vh;
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  h2 {
+    font-size: 24px;
+    line-height: 1.45;
+  }
+`;
+
+const RenderAccessFromSUSI = ({ pathname }) => {
+  return (
+    <Container>
+      <a href={urls.HOME_URL}>
+        <SusiLogo />
+      </a>
+      <h2>
+        Access to this page is restricted from local device,
+        <br />
+        try accessing from live website
```

Comment:
Make channge here as suggested in previous review.

Reply:
Oh, sorry